Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemoteTiledBacking.cpp

```diff
@@ -36,7 +36,7 @@ using namespace WebCore;
 
 PlatformCALayerRemoteTiledBacking::PlatformCALayerRemoteTiledBacking(LayerType layerType, PlatformCALayerClient* owner, RemoteLayerTreeContext& context)
     : PlatformCALayerRemote(layerType, owner, context)
-    , m_tileController(makeUnique<TileController>(this))
+    , m_tileController(makeUnique<TileController>(this, false))
```

Comment:
Boolean trap here (hard to know what `false` means). Would be good to either add a comment, or use an enum.

Reply:
Changed to use enum.